db/migrate/20230115233416_create_sidebar_sections.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateSidebarSections < ActiveRecord::Migration[7.0]
+  def change
+    create_table :sidebar_sections do |t|
+      t.integer :user_id
+      t.string :title, unique: true
+      t.timestamps
+    end
+
+    add_index :sidebar_sections, :user_id
```

Comment:
```suggestion
```

I don't think this is required because the index on `user_id` and `title` should cover queries filtering by `user_id`.